### DISCUSSION TITLE
Error downloading kernel_4.4.271-5590_08.00-ADv1.patch

### DISCUSSION THREAD

**Discussion by ACEx86** [Freetz-NG/freetz-ng#1083] [Allgemein] [answered]
I get an error message when make is trying to download the kernel patch
Im trying to build for 5590 and all the mirrors fail.

tools/freetz_download dl kernel_4.4.271-5590_08.00-ADv1.patch.xz @MIRROR/ 76bfb772beaa3c126a0a61b567fd369d024e1ffa7976accabd49cc1542e33189 

--2024-12-24 15:16:20--  https://freetz-ng.root.sx/kernel_4.4.271-5590_08.00-ADv1.patch.xz
Resolving freetz-ng.root.sx... 0.0.0.0, ::
Connecting to freetz-ng.root.sx|0.0.0.0|:443... failed: Connection refused.
Connecting to freetz-ng.root.sx|::|:443... failed: Connection refused.
Download failed - "https://freetz-ng.root.sx/kernel_4.4.271-5590_08.00-ADv1.patch.xz"  ->  error code 4

**Comment by fda77** [comment, marked as answer]
Your name resolution may be broken, the A record"0.0.0.0" is not correct and the AAAA record empty
